core-bundle/src/Resources/config/services.yml

```diff
@@ -336,6 +336,17 @@ services:
         tags:
             - { name: contao.picker_provider }
 
+    contao.picker.article_content_provider:
```

Comment:
We have agreed on using the class name as service key for all new services, so this needs to become `Contao\CoreBundle\Picker\ArticleContentPickerProvider`.